packages/build/src/error/parse/plugin.ts

```diff
@@ -36,27 +36,32 @@ const getVersion = function ({ version }) {
   return version
 }
 
-export const getHomepage = function (pluginPackageJson = {}, { loadedFrom } = {}) {
+type pkgJSONData = { name?: string; bugs?: { url?: string }; repository?: { url?: string } }
```

Comment:
Tried to leverage `read-pkg-up`'s PackageJson type:
- https://github.com/netlify/build/blob/481a42e210c9f33176f70dfbfc95b84cbe65f1b0/packages/build/src/utils/package.ts#L6

Which I believe is the original source of this information. Unfortunately couldn't get it to work and just tossed in the towel.

Reply:
Have you though about using interesection types here?
Given that we end up using the different parts (name, bugs, repository) in the `get*` functions below, maybe it's worth creating specific types for those and then and intersection one for `pkgJSONData`?

So something like
```
type pkgJSONName = { name?: string }
type pkgJSONBugs = { bugs?: { url?: string } }
type pkgJSONRepo = { repository?: { url?: string } }

type pkgJSONData = pkgJSONName & pkgJSONBugs & pkgJSONRepo
```

And then using the individual "sub-types" in the methods below. Not sure if an improvement or not.